Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -21,7 +21,6 @@ namespace AzToolsFramework::ViewportUi::Internal
 {
     const static int HighlightBorderSize = 5;
     const static char* HighlightBorderColor = "#4A90E2";
```

Comment:
Nit: I know you didn't change this, but `HilightBorderColor` is non-const and globally accessible, we should add a `const` to the `*`:
```suggestion
    const static const char* const HighlightBorderColor = "#4A90E2";
```

Reply:
D'oh, I did the suggestion wrong, I added one too many `const`s:
```suggestion
    const static char* const HighlightBorderColor = "#4A90E2";
```